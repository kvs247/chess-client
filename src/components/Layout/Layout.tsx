import { FC } from "react";
import { ConfigProvider } from "antd";
import { LayoutProps } from "./types";

const Layout: FC<LayoutProps> = ({ children }) => {
  return (
    <ConfigProvider
      theme={{
        token: {
          colorPrimary: "#00b96b",
          borderRadius: 100,
          colorBgContainer: "grey",
        },
      }}
    >
      <div>{children}</div>
    </ConfigProvider>
  );
};

export default Layout;