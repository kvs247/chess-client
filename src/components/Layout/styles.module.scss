.Layout {
  height: 100vh;
  width: 100vw;
  background: grey;
  overflow: hidden;
}