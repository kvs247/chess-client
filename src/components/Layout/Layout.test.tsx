import { render } from "@testing-library/react";
import Layout from "./Layout";

it("renders", () => {
  const { asFragment } = render(
    <Layout>
      <div>test</div>
    </Layout>
  );
  expect(asFragment()).toMatchSnapshot();
});