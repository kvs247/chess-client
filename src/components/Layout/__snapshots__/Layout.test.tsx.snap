// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <div>
      test
    </div>
  </div>
</DocumentFragment>
`;
